Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -645,6 +669,24 @@ open class ChatMessageContentView: _View, ThemeProvider {
     @objc open func handleTapOnDeliveryStatusView() {
         delegate?.messageContentViewDidTapOnDeliveryStatusIndicator(indexPath?())
     }
+    
+    // MARK: - UITextViewDelegate
+    
+    open func textView(_ textView: UITextView, shouldInteractWith URL: URL, in characterRange: NSRange, interaction: UITextItemInteraction) -> Bool {
+        if let mentionedUsers = content?.mentionedUsers, !mentionedUsers.isEmpty {
```

Comment:
guard?

Reply:
The normal branch of this logic is to actually open a URL, if we in the future introduce more logic here, it will be easy to introduce at the bottom. The early return pattern here doesn't make much sense to me IMO